### ISSUE TITLE
Template repo not found!

### ISSUE THREAD

**Issue by pr13260** [Aynh/cloudflare-gdrive#3]
Is there any guide on how to use it ?

**Comment by aynh** [comment]
Yes, it's there in the [README](https://github.com/aynh/cloudflare-gdrive/blob/main/README.md#setup). The template doesn't matter, just create your own using `wrangler init`.

There is also an _incomplete_ [next branch](https://github.com/aynh/cloudflare-gdrive/tree/next) in case you want to check.
